.ci/azure-pipelines/build-ubuntu.yaml

```diff
@@ -1,22 +1,50 @@
 resources:
   containers:
+    - container: env1604
+      image: pointcloudlibrary/env:16.04
     - container: env1910
       image: pointcloudlibrary/env:19.10
 
 jobs:
-  - job: ubuntu1910
-    displayName: Ubuntu 19.10
-    timeoutInMinutes: 0
+  - job: ubuntu
+    displayName: Ubuntu
     pool:
       vmImage: 'Ubuntu 16.04'
-    container: env1910
+    timeoutInMinutes: 0
+    strategy:
+      matrix:
+#        ubuntu16 Clang:
+#          containerResource: env1604
+#          CC: clang
+#          CXX: clang++
+#          CMakeArgs: '-DPCL_WARNINGS_ARE_ERRORS=ON'
+#
+#        ubuntu19 Clang:
+#          containerResource: env1910
+#          CC: clang
+#          CXX: clang++
+#          CMakeArgs: ''
+
+        ubuntu16 GCC:
```

Comment:
```suggestion
        16.04 GCC:
```

1. Let's have the full version, 16 or 19 is ambiguous.
2. "ubuntu" in the name is superfluous because it is already in the `displayName`

![image](https://user-images.githubusercontent.com/1241736/77529408-d61fac00-6e8f-11ea-831c-84e5f4555c3a.png)